lib/bundles/inspec-habitat/profile.rb

```diff
@@ -346,16 +352,15 @@ def run_hook_contents
 
 while true; do
   echo "Executing InSpec for ${PROFILE_IDENT}"
-  inspec exec {{pkg.path}}/dist --format=json > ${RESULTS_FILE} 2>${ERROR_FILE}
+  inspec exec {{pkg.path}}/dist --format=json > ${RESULTS_FILE}
```

Comment:
Now that we're deleting this can you also remove https://github.com/chef/inspec/pull/1826/files#diff-93d8a0fc1b64e361ffb0725d8ed10a37L342